packages/vaadin-combo-box/src/vaadin-combo-box-item.js

```diff
@@ -164,18 +146,12 @@ class ComboBoxItemElement extends ThemableMixin(DirMixin(PolymerElement)) {
     }
   }
 
-  _updateLabel(label, _itemTemplateInstance) {
-    if (_itemTemplateInstance === undefined && this.$.content && !this.renderer) {
-      // Only set label to textContent no template
-      this.$.content.textContent = label;
-    }
-  }
+  _updateLabel(label) {
+    if (this.renderer) return;
```

Comment:
Polymer multi-property observers typically need passing all the arguments, so I would change this:

```suggestion
  _updateLabel(label, renderer) {
    if (renderer) return;
```

Reply:
Fixed.